common/djangoapps/student/forms.py

```diff
@@ -301,9 +301,7 @@ def clean_email(self):
                     raise ValidationError(_("Unauthorized email address."))
         if email_exists_or_retired(email):
             raise ValidationError(
-                _(
-                    "It looks like {email} belongs to an existing account. Try again with a different email address."
-                ).format(email=email)
+                _("It looks like this email was already registered. Try again with a different email address.")
```

Comment:
In most places you didn't remove the email address format thing (`{email}`) from the string, why did you do it here?

Reply:
There were some things that looked like these that got changed and then changed back as I figured out the various ways we handle these messages, so it's probably copypasta from when I was sorting that all out. I'll add it back and fix up whatever tests got broken along the way.